scripts/utils.js

```diff
@@ -288,21 +320,60 @@ function buildTagsBlock() {
       recBlock.before(tagsBlock);
     }
   } else {
-    main.lastElementChild.append(tagsBlock);
+    mainEl.lastElementChild.append(tagsBlock);
   }
 }
 
+async function buildArticleMeta(mainEl) {
+  const miloLibs = getLibs();
+  const { getMetadata, getConfig } = await import(`${miloLibs}/utils/utils.js`);
+
+  // author + publication date
+  const author = getMetadata('author') || 'Adobe Communications Team';
+  const { codeRoot } = getConfig();
+  const authorURL = getMetadata('author-url') || (author ? `${codeRoot}/authors/${author.replace(/[^0-9a-z]/gi, '-').toLowerCase()}` : null);
+
+  const publicationDate = getMetadata('publication-date');
+
+  const articleMeta = buildBlock('article-meta', [
+    [`<p>${authorURL ? `<a href="${authorURL}">${author}</a>` : author}</p>
+      <p>${publicationDate}</p>`],
+  ]);
+
+  // put article meta (author + link sharings) in front of first content div
+  const allContentDivs = mainEl.querySelectorAll(':scope > div');
+  if (!allContentDivs) return;
+
+  const contentBlock = [...allContentDivs].find((div) => !div.querySelector('.article-hero-banner') && div.querySelector('p'));
+  if (!contentBlock) return;
+
+  contentBlock.insertBefore(articleMeta, contentBlock.firstChild);
+}
+
+async function addContentTypeToMainClass(mainEl) {
+  const miloLibs = getLibs();
+  const { getMetadata } = await import(`${miloLibs}/utils/utils.js`);
+  const pageText = getMetadata('content-type');
+  const pageClass = `page-${pageText}`;
```

Comment:
It's used mainly for styling the hero `masonry-layout` block in home page in the `styles.css`. I can add a class to the block directly instead of using page scope styling.

Reply:
Removed  `addContentTypeToMainClass()` from `utils.js` and added `hero-masonry` class for specific styling on `masonry-layout` block instead. Let me know if that works / needs to updated further, thank you